cactusbot/commands/command.py

```diff
@@ -1,3 +1,5 @@
+# pylint: skip-file
```

Comment:
Yeah... looking back on it, that's a bit too much cheating. Switching to individual errors. :+1: